### PR TITLE
Amazon Test Cases

### PR DESCRIPTION
**Pull request by selmanHa**
60 Amazon Test cases are done, all are passed and good to go. Everyone should be able to review the tests to add your parts. Good luck

### CODE REVIEW

**Comment by selmanHa** [comment]
Reviewed.

File: Generic/src/main/java/common/CommonAPI.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
     @BeforeMethod
     public void setUp(String url) {
 
-        System.setProperty("webdriver.chrome.driver", "/Users/sams/Desktop/teamWork/WebAutomationTeam1/Generic/web-driver/chromedriver");
```
Comment (selmanHa) [inline Generic/src/main/java/common/CommonAPI.java]: This is added by mistake, need to be deleted...

**Comment by bare612** [comment]
added amazon

**Comment by bare612** [comment]
This what I mean

**Pull request closed without merging**